rofi-pass

```diff
@@ -417,7 +417,11 @@ mainMenu () {
 	fi
 	if [[ -z "${stuff["${USERNAME_field}"]}" ]]; then
 		if [[ -n $default_user ]]; then
-			stuff["${USERNAME_field}"]="${default_user}"
+		    if [[ "$default_user" == "%filename" ]]; then
+		      stuff["${USERNAME_field}"]="$(basename $selected_password)"
+		    else
+    			stuff["${USERNAME_field}"]="${default_user}"
+    	  fi
```

Comment:
Just as a small note.
Those changes introduced a mixup of real tabs and spaces, so the formatting is slightly off